frontend/src/styles/_uswds-theme-custom-styles.scss

```diff
@@ -30,3 +30,8 @@ i.e.
     @include u-shadow(4);
   }
 }
+@include at-media("desktop") {
```

Comment:
Imagine you've tried some things out, but is there a way to avoid this? Can we mirror styling of the navbar items, or is there something throwing this off that is too difficult to deal with?

Reply:
There is no 5px margin from [USWDS](https://designsystem.digital.gov/utilities/margin-and-padding/). The extra pixel did make a difference IMHO.